cupy/lib/__init__.py

```diff
@@ -1,8 +0,0 @@
-from cupy.lib import stride_tricks  # NOQA
```

Comment:
It is a documented interface, so it should not be removed.

Reply:
Fixed!